age/keysource.go

```diff
@@ -225,6 +225,11 @@ func (key *MasterKey) ToMap() map[string]interface{} {
 	return out
 }
 
+// TypeToString converts key type to a string
+func (key *MasterKey) TypeToString() string {
+	return "age"
```

Comment:
This could be exported as a `const`, making it programmatically easier to check against.